matrix/main.py

```diff
@@ -44,7 +45,10 @@ def setup(matrix, args=None):
     parser.add_argument("-i", "--interval", default=5.0, type=float)
     parser.add_argument("-p", "--path", default=Path.cwd() / "tests",
                         type=Path)
-    parser.add_argument("config_file", type=open)
+    parser.add_argument("-c", "--config-file",
+                        default=resource_filename(__name__, 'matrix.yaml'))
```

Comment:
Doesn't this favor thinking about the config file as residing as part of the matrix checkout? wouldn't cwd()/tests/matrix.yaml or just ./matrix.yaml be a better choice? Also does this interfere with the #!matrix style used in some of the tests? I don't know that we need to support that mode of operations but it would make having a single exec that BT can call simple.

If this should only represent the default tests then we might go about this another way, config_fiiles with type list(path) where the default includes the resource local one?